export/pdf-component.js

```diff
@@ -59,7 +59,16 @@ class PrintToPdf extends LitElement {
       html2pdf().set({
         html2canvas: {
           scrollX: 0,
-          scrollY:0
+          scrollY:0,
+          scale: 2
+        },
+        image: {
+          type: 'jpeg',
+          quality: 1
+        },
+        jsPDF: {
+          unit: 'in',
```

Comment:
I would use mm, inches?
Does improve substantially with scale to 2?

Reply:
let me try with mm

Reply:
just with the scale:2 it is enough for get better resulution